indigo/distribution.yaml

```diff
@@ -1,6 +1,6 @@
 %YAML 1.1
 # ROS distribution file
-# see REP 143: http://ros.org/reps/rep-0143.html
+# see REP 141: http://ros.org/reps/rep-0141.html
```

Comment:
You need to update your Python rosdistro package. Your version is still using an older REP.